docs/releasing/publishing-a-pre-release.md

```diff
@@ -1,19 +1,33 @@
-# Publishing pre-release of GOV.UK Frontend
+# Publish pre-release of GOV.UK Frontend
 
-We use pre-releases when:
-- We are working on a new contribution and want to see it in the Design System website
-- We want to try an experimental feature as if it's pushed to npm.
+This pre-release guidance is aimed at Design System team members. If you're an external contributor who needs to create a pre-release, please [contact the Design System team](https://design-system.service.gov.uk/get-in-touch/) and we'll do it for you.
 
-This is done by pushing the files used for a GOV.UK Frontend release (the contents of the `package` directory) to a new branch which can be installed by npm as if it was a released npm package.
+Before you use a pre-release, you need to make changes to GOV.UK Frontend. Then follow these instructions.
```

Comment:
```suggestion
Before you publish a pre-release, you need to have committed a code change to GOV.UK Frontend. Then follow these instructions.
```

Reply:
@hannalaakso The pre-i reviewer liked the change you suggested. I've now sent the content for 2i.